src/main/java/com/minecolonies/blockout/util/Constants.java

```diff
@@ -0,0 +1,11 @@
+package com.minecolonies.blockout.util;
+
+public class Constants
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1118)